IdentityCore/src/MSIDError.h

```diff
@@ -356,7 +356,10 @@ typedef NS_ENUM(NSInteger, MSIDErrorCode)
     
     // Error is thrown when PSSO user registration attempted with no biometrics available and sekey biometric policy is configured
     MSIDErrorPSSOBiometricsNotAvailable        =   -51845,
-
+    
+    // Error is thrown when a default account cannot be determined because zero or multiple accounts exist.
+    MSIDErrorNoUniqueAccountFound = -51846,
```

Comment:
we need to update errors mapping MSAL as well. Also do we need this error at all? Should we throw "interaction required" if we didn't find "default" account?